plugin.py

```diff
@@ -129,13 +136,27 @@ def additional_variables(cls) -> Optional[Dict[str, str]]:
             "launcher_version": launcher_version
         }
 
+    @classmethod
+    def on_pre_start(cls, window: sublime.Window, initiating_view: sublime.View,
+                     workspace_folders: List[WorkspaceFolder], configuration: ClientConfig) -> Optional[str]:
+        javaagent_arg = "-javaagent:" + lombok_path(cls.storage_subpath())
+        # Prevent adding the argument multiple times
```

Comment:
I don't think the case when argument is added multiple times should be possible as `on_pre_start` should be always called with freshly created `config`. Did you see it happening when testing?

Reply:
`Troubleshoot Server` command does add the argument too.